pydata/templates/pydata/person_create_form.html

```diff
@@ -0,0 +1,5 @@
+{% extends "pydata/generic_import_form.html" %}
+
+{% block modal %}
+{% include "pydata/person_import_from_url.html" %}
```

Comment:
I'd call this template "person_import_modal" -- had I been wiser I'd call the similar template in `workshops` this way too.